### PR TITLE
Add Multi Conditional Endorsement Triple

### PR DESCRIPTION
**Pull request by andrew-draper**

### CODE REVIEW

File: cddl/multi-conditional-endorsement-triple.cddl [hunk 1]
```diff
@@ -0,0 +1,4 @@
+multi-conditional-endorsement-triple-record = [
+  cond: [ + stateful-environment-record],
```
Comment (yogeshbdeshpande) [inline cddl/multi-conditional-endorsement-triple.cddl]:
```suggestion
  cond: [ + stateful-environment-record], ; a series of stateful environments to be matched against evidence 
```

File: cddl/multi-conditional-endorsement-triple.cddl [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  end:  [ + stateful-environment-record]
```
Comment (yogeshbdeshpande) [inline cddl/multi-conditional-endorsement-triple.cddl]:
```suggestion
  end:  [ + stateful-environment-record] ; a series of endorsements to be accepted upon matching the subject `cond`
```

File: draft-ietf-rats-corim.md [hunk 3]
```diff
@@ -1249,6 +1253,26 @@ applies to all measurements in the triple, including those in `measurement-value
 {::include cddl/conditional-endorsement-triple-record.cddl}
 ~~~
 
+#### Multi Conditional Endorsement Triple {#sec-comid-triple-multi-cond-end}
+
+A Multi Conditional Endorsement Triple behaves similarly to the Conditional Endorsement Triple {{sec-comid-triple-cond-end})
```
Comment (yogeshbdeshpande) [inline draft-ietf-rats-corim.md:1258]: I think the first statement is not strictly true if you revisit the definition of conditional-endorsement-triple-record. There the First reference value matching stop the matching process. However in this triple, we expect to match All the Subject conditions prior to accepting the series.

**Comment by andrew-draper** [comment]
This is premature, dropping for now

**Pull request closed without merging**